src/fprime/cookiecutter_templates/cookiecutter-fprime-project/hooks/post_gen_project.py

```diff
@@ -48,9 +48,10 @@
     print(f"[ERROR] Unable to checkout tag: {latest_tag_name}. Exit...")
     sys.exit(1)  # sys.exit(1) indicates failure to cookiecutter
 
-# Checkout submodules (e.g. googletest)
+# Checkout F´ submodules (e.g. googletest)
 res = subprocess.run(
     ["git", "submodule", "update", "--init", "--recursive"],
+    cwd="./fprime",
```

Comment:
Why?  Wouldn't a recursive update at the higher level catch fprime?

Reply:
I concur; running `git submodule update --init --recursive` at the project level should already initialize all nested submodules.

Reply:
The issue was that `git submodule update` was updating the `fprime/` submodule of the (top level) project repository, which was causing the bug of `fprime/` going back to `devel` instead of latest release. Reordering will also fix the issue